app-backend/db/src/main/scala/filters/Filters.scala

```diff
@@ -177,7 +177,7 @@ object Filters {
       putGeom: Put[Projected[Polygon]]): List[Option[Fragment]] =
     List(
       taskQP.status.toList.toNel map { qp =>
-        Fragments.in(fr"status", qp)
+        Fragments.in(fr"tasks.status", qp)
```

Comment:
Both `annotation_projects` and `tasks` have a `status` field and are in the query so we need to be specific.